astro/src/content/docs/operate/deploy/terraform.mdx

```diff
@@ -0,0 +1,432 @@
+</Aside>
+
+Now that you've configured Terraform to handle some of FusionAuth's default configuration elements, let's walk through creating, updating, and removing other resources.
+
+## Creating Resources
+
+To create a new resource review the list of resources available to you in the [FusionAuth Terraform Provider documentation](https://registry.terraform.io/providers/fusionauth/fusionauth/latest/docs). Pick the resource you're interested in. Each resource contains information about required and optional arguments.
+
+For this example, you'll create the following:
+
+* A new tenant called `Forum`.
+* An application living in that new tenant, also called `Forum`.
+* Two roles for that application, `admin` and `user`.
+* A token signing key for that application.
+* Email templates for password emails that will be assigned to the tenant.
+
+### Variables
+
+You'll need to update `variables.tf` to include additional variables.
+
+<RemoteCode lang="hcl" title="Forum Variable" url="https://raw.githubusercontent.com/FusionAuth/fusionauth-example-terraform/main/examples/create/variables.tf" />
+
+Next, update the `terraform.tfvars` with customizations of those values. It should look similar to the below file, but you'll need to update it.
+
+<RemoteCode lang="hcl" title="Forum Variable Definitions" url="https://raw.githubusercontent.com/FusionAuth/fusionauth-example-terraform/main/examples/create/terraform.tfvars" />
+
+* Set the `fusionauth_api_key` variable to the value of the previously created API key.
+* Set the `fusionauth_host` variable to the hostname of the FusionAuth instance you want to manage. Make sure this value is the full URL including the protocol of your FusionAuth instance. When using docker, this will typically be `http://localhost:9011`.
+* The `fusionauth_default_tenant_id` variable should be the UUID of the default tenant.
+
+### The Terraform File
+
+Next, update the `main.tf` file.
+
+If you haven't worked through the prerequisites above, make sure you add the `required_providers` and `provider` sections.
+
+<RemoteCode lang="hcl" title="main.tf Initial Sections" url="https://raw.githubusercontent.com/fusionauth/fusionauth-example-terraform/main/examples/import/main.tf" tags="terraformProvider" />
+
+Then, configure the `Forum` tenant; this tenant will hold your users and applications.
+
+<RemoteCode lang="hcl" title="Creating A Forum Tenant" url="https://raw.githubusercontent.com/fusionauth/fusionauth-example-terraform/main/examples/create/main.tf" tags="createForumTenant" />
+
+Then, configure the signing key. This will be used to sign access tokens for the `Forum` application.
+
+<RemoteCode lang="hcl" title="Creating A Signing Key" url="https://raw.githubusercontent.com/fusionauth/fusionauth-example-terraform/main/examples/create/main.tf" tags="createKey" />
```

Comment:
Maybe it was a build time issue, but this rendered as "not available" inside the block.

Reply:
Worked for me. Suspect it was a build time issue.